hipscat/catalog.py

```diff
@@ -44,321 +43,424 @@ def __init__(self, catname='gaia', source='local', location='/epyc/projects3/sam
-                fs_clean = file_source
-                if fs_clean[-1] != '/':
-                    fs_clean += '/'
-                urls = glob.glob('{}*{}'.format(fs_clean, fmt))
-            else:
-                sys.exit('Local files not found at source {}'.format(file_source))
+            urls = glob.glob("{}*{}".format(fs_clean, fmt))
 
         if limit:
             urls = urls[0:limit]
 
         if verbose:
-            print(f'Attempting to format files: {len(urls)}')
+            print(f"Attempting to format files: {len(urls)}")
 
         if len(urls):
-            self.partitioner = pt.Partitioner(catname=self.catname, fmt=fmt, urls=urls, id_kw=id_kw,
-                        order_k=10, verbose=verbose, debug=debug, ra_kw=ra_kw, dec_kw=dec_kw, 
-                        location=self.location)
+            self.partitioner = Partitioner(
+                catname=self.catname,
+                fmt=fmt,
+                urls=urls,
+                id_kw=id_kw,
+                order_k=10,
+                verbose=verbose,
+                debug=debug,
+                ra_kw=ra_kw,
+                dec_kw=dec_kw,
+                location=self.location,
+            )
 
             if debug:
                 self.partitioner.gather_statistics()
-                self.partitioner.compute_partitioning_map(max_counts_per_partition=threshold)
+                self.partitioner.compute_partitioning_map(
+                    max_counts_per_partition=threshold
+                )
                 self.partitioner.write_structure_metadata()
             else:
                 self.partitioner.run(client=client)
                 self.__init__(self.catname, location=self.location)
         else:
-            print('No files Found!')
-
+            raise FileNotFoundError("No files Found!")
 
     def cone_search(self, ra, dec, radius, columns=None):
-        '''
+        """
         Perform a cone search on HiPSCat
 
-        Parameters:
+        Args:
          ra:     float, Right Ascension
          dec:    float, Declination
          radius: float, Radius from center that circumvents, must be in degrees
-        '''
-        assert self.hips_metadata is not None, f'{self} hipscat metadata not found. {self}.hips_import() needs to be (re-) ran'
-        assert isinstance(ra, (int, float)), f'ra must be a number'
-        assert isinstance(dec, (int, float)), f'dec must be a number'
-        assert isinstance(radius, (int, float)), f'radius must be a number'
-        
-        #establish metadata for the returning dask.dataframe
+         Returns:
+             Dask dataframe containing the data within the cone_search
+        """
+        if self.hips_metadata is None:
+            raise ValueError(
+                f"{self} hipscat metadata not found. {self}.hips_import() needs to be (re-) ran"
+            )
+
+        if not isinstance(ra, (int, float)):
+            raise TypeError(f"ra must be a number")
```

Comment:
Here and next few lines, there's no string formatting happening, so you can drop the f.